book.toml

```diff
@@ -4,19 +4,33 @@ authors = ["the rust-unofficial authors"]
 description = "A catalogue of Rust design patterns, anti-patterns and idioms"
 language = "en"
 multilingual = false
-src = "."
+src = "src"
 
 [build]
 create-missing = false
+extra-watch-dirs = ["po", "third_party"]
+
+[preprocessor.gettext]
+after = ["links"]
 
 [rust]
 edition = "2018"
 
 [output.html]
+curly-quotes = true
 default-theme = "rust"
 site-url = "/patterns/"
 git-repository-url = "https://github.com/rust-unofficial/patterns"
 git-repository-icon = "fa-github"
 edit-url-template = "https://github.com/rust-unofficial/patterns/edit/main/{path}"
+additional-css = ["./language-picker.css"]
+additional-js = ["./third_party/mdbook/book.js"]
+
+[output.html.fold]
+enable = true
+level = 1
+
+[output.html.playground]
+editable = false
```

Comment:
why you disabled editable?

Reply:
> It's live editing on the website

what do you mean?
What's the difference between having this setting true or false?